ui/plugins/eu.esdihumboldt.hale.ui.views.schemas/src/eu/esdihumboldt/hale/ui/views/schemas/explorer/ServiceSchemaExplorer.java

```diff
@@ -117,6 +128,7 @@ public void run() {
 						if (spaceID.equals(schemaSpace)) {
 							explorer.setSchema(schemaService.getSchemas(spaceID));
 						}
+						refreshInDisplayThread();
```

Comment:
There are a lot of places where this is added - was it really that many cases where UI updates did not work before properly?

Reply:
In the case of ResourcesView, yes it didn't work for many use cases. 
For the SchemaExplorer, `refreshInDisplayThread()` was called at some places so I added it in the missing ones.

Reply:
I think for instance in this place, because `setSchema` is called, which changes the input of the viewer completely, it is not necessary.